client/src/components/Trip.js

```diff
@@ -34,7 +41,7 @@ const Trip = () => {
         <div>
             <h2>Holiday Finder</h2>
             <TripForm getTrip={getTrip} />
-            {!holidays ? (<p>Find holidays for your trip!</p>) : (<Holiday days={holidays}/>)}
+            {!holidays ? (<p>{message}</p>) : (<Holiday days={holidays} />)}
```

Comment:
Consider switching this logic so you're not testing a negative
```suggestion
            {holidays ? (<Holiday days={holidays} />) : (<p>{message}</p>)}
```